docs/contrib_components/api/usage.rst

```diff
@@ -0,0 +1,671 @@
+Wagtail API usage guide
+=======================
+
+Listing views
+-------------
+
+Performing a ``GET`` request against one of the endpoints will get you a listing of objects in that endpoint. The response will look something like this:
+
+.. code-block:: json
+
+    GET /api/v1/endpoint name/
```

Comment:
Very pedantic, but this is not a valid GET request line, as the ' ' space char needs to be URL encoded. Perhaps use a hyphen to avoid any ambiguity?